.github/actions/godot-build/action.yml

```diff
@@ -59,10 +53,6 @@ runs:
           fi
         fi
 
-        if [ "${{ inputs.mingw-prefix }}" != "" ]; then
-          scons platform=${{ inputs.platform }} target=${{ inputs.target }} tests=${{ inputs.tests }} ${{ env.SCONSFLAGS }} use_mingw=yes
-        else
-          scons platform=${{ inputs.platform }} target=${{ inputs.target }} tests=${{ inputs.tests }} ${{ env.SCONSFLAGS }}
-        fi
+        scons platform=${{ inputs.platform }} target=${{ inputs.target }} tests=${{ inputs.tests }} ${{ env.SCONSFLAGS }} "cache_path=${{ inputs.scons-cache }}" cache_limit=${{ inputs.scons-cache-limit }}
```

Comment:
Whats up with cache_path?

Reply:
I've seen it used in another build repo for windows build, it's basically same as SCONS_CACHE env var but as I understand on windows that isn't taken. So this makes sure it's taken (As it explicitly uses it). The var is defined in the blazium-engine/blazium repo at SConstruct file line 223.

```cpp
opts.Add("cache_path", "Path to a directory where SCons cache files will be stored. No value disables the cache.", "")
```